lib/rubocop/cop/rails/use_time_zone.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of Time.zone= method.
+      # Built on top of this nice Thoughtbot article (https://thoughtbot.com/blog/its-about-time-zones)
+      #
+      # @example
+      #   # bad
+      #   Time.zone = 'EST'
+      #
+      #   # good
+      #   Time.use_zone('EST', &block)
+      #
+      class UseTimeZone < Base
+        MSG = 'Do not use `Time.zone=` for time zone sensitive code. Use `Time.use_zone(time_zone, &block)` instead.'
```

Comment:
Can you simplify the message?

```suggestion
        MSG = 'Use `Time.use_zone` with blocks instead of `Time.zone=`.'
```